dts/vendor/nordic/nrf54l15_partition.dtsi

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&cpuapp_rram {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		boot_partition: partition@0 {
+			label = "mcuboot";
+			reg = <0x0 DT_SIZE_K(64)>;
+		};
+
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x10000 DT_SIZE_K(648)>;
+		};
+
+		slot1_partition: partition@b2000 {
+			label = "image-1";
+			reg = <0xb2000 DT_SIZE_K(648)>;
+		};
+
+		/* 32KiB from 0x154000 to 0x15bfff reserved for TF-M partitions */
```

Comment:
This comment now seems invalid since this is not the ns target. Maybe its bettter to remove the comment and use the 32KiB in the other partitions.